File: ChurchService.ng/src/app/church/church.model.ts
export class MemberDetailsModel {
        
    public UserId: string;
    public FullName: string;
    public Email: string;
    public Phone: string;

}

export class AddMemberModel {

    public FirstName: string;
    public LastName: string; 
    public Email: string; 
    public PhoneNumber: string; 
    public Address: string; 
    public City: string; 
    public State: string; 
    public Zip: string; 
    public Password: string; 
    
}

export class ChurchModel {

    public Name: string;
    public Email: string;
    public PhoneNumber: string;
    public Address: string;
    public City: string;
    public State: string;
    public Zip: string;
    public ChurchLogoDataUrl: string;
    public ChurchLogoMime: string;

}

File: ChurchService/ChurchService.Library/Models/Churches/ChurchModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ChurchService.Library.Models.Churches
{
    public class ChurchModel
    {
        public Guid ChurchId { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
        public string PhoneNumber { get; set; }
        public string Address { get; set; }
        public string City { get; set; }
        public string State { get; set; }
        public string Zip { get; set; }
        public string ChurchLogoDataUrl { get; set; }
        public string ChurchLogoMime { get; set; }
    }
}
